src/main/kotlin/com/github/jyoo980/reachhover/model/ReachabilityHoverContext.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.jyoo980.reachhover.model
+
+import com.intellij.psi.PsiElement
+import com.intellij.ui.awt.RelativePoint
+
+data class ReachabilityHoverContext(val elementToInspect: PsiElement, val location: RelativePoint) {
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
I can take a look, that sounds reasonable.

I generated this overwritten implementation of `equals` since I was concerned about the autogenerated data class methods also taking into account the value of `RelativePoint`.

Reply:
So yeah, it looks like having the autogenerated methods don't really work (I get more than one popup created for the same hover action).

If we want to stay away from overriding equals, I guess I could just take two contexts and check if their `elementToInspect` fields are equal. What are your thoughts?

Reply:
@jyoo980 Yea, I'd prefer comparing their fields over overriding `equals` in that case. Such `equals` could even be more confusing, since it's not some expected comparison logic for a general data class.

Reply:
Done in #6.

Oleg! Thank you, get some sleep! I'll chat with you again next week. Take care.